modules/collect/src/main/java/com/opengamma/collect/io/IniFile.java

```diff
@@ -84,6 +108,56 @@ public static IniFile of(CharSource source) {
     }
   }
 
+  //-------------------------------------------------------------------------
+  /**
+   * Returns a single INI file that is the chained combination of the inputs.
+   * <p>
+   * A chained INI file must have one special section, 'chain', with three properties.
+   * The first is 'priority', an integer, which allows the input files to be sorted.
+   * The second is 'nextInChain', a boolean, which determines where to chain on to the next file or not.
```

Comment:
I think ``nextInChain`` is a potentially confusing name. My initial assumption was that it is referring to the file that contains it. i.e. this file is the next file in the chain. Which clearly doesn't make sense. How about ``enableChaining`` or ``chainNextFile``?